pleskdistup/messages.py

```diff
@@ -59,6 +59,6 @@
 REBOOT_WARN_MESSAGE = """\r\033[93m****************************** WARNING ***********************************************
```

Comment:
Where is it used?

Reply:
I am passing this message to the `PreRebootPause` action. However it could take any other message as well.